test/e2e/storage/drivers/csi.go

```diff
@@ -107,6 +107,7 @@ func (h *hostpathCSIDriver) CreateDriver() {
 		OldDriverName:            h.driverInfo.Name,
 		NewDriverName:            GetUniqueDriverName(h),
 		DriverContainerName:      "hostpath",
+		DriverContainerArguments: []string{"--drivername=csi-mock-" + f.UniqueName},
```

Comment:
drivername should be csi-hostpath?

Reply:
Of course, I forgot to push one extra commit.